source/client-side-encryption/tests/aggregate.template.yml

```diff
@@ -0,0 +1,61 @@
+runOn:
```

Comment:
Add a test for db/client level aggregate.  It should fail, iirc

Reply:
Added a failing db aggregate test.

Reply:
What's the name of that test? I'm not finding it

Reply:
Last test in this file, "Database aggregate should fail"

Reply:
Ack